contrib/helm/harbor/templates/ingress/secret.yaml

```diff
@@ -1,11 +1,15 @@
+{{ if not .Values.insecureRegistry }}
+{{ $ca := genCA "harbor-ca" 365 }}
```

Comment:
we should only do this if `.Values.tlsCrt`, `.Values.tlsKey`, `.Values.tlsCrt` is not set.

Reply:
The code below uses `.Values.tlsCrt` if it's set by users, and uses the generated values if `.Values.tlsCrt` is not set.

  tls.crt: {{ .Values.tlsCrt | default $cert.Cert | b64enc | quote }}
  tls.key: {{ .Values.tlsKey | default $cert.Key | b64enc | quote }}
  ca.crt: {{ .Values.caCrt | default $ca.Cert | b64enc | quote }}

Reply:
I understand, but we shouldn't generate certs if they're not needed, unless the logic to not do so is too complicated to do in a helm template.